python-client/pyproject.toml

```diff
@@ -151,6 +151,7 @@ dependencies = [
     "chardet", # text metadata
     "jinja2>=3", # scan template (temporary)
     "requests-toolbelt>=0.9.1",
+    "detoxify>=0.5.0",
```

Comment:
It should not be included here, but in the `llm` group.

Reply:
DONE